correctness/docker-compose.yml

```diff
@@ -0,0 +1,32 @@
+services:
+  mockmetheus:
+    container_name: mockmetheus
+    build: .
+    volumes:
+      - .:/app
+    working_dir: /app
+    command: ["sleep", "infinity"]
+    environment:
+      AWS_ACCESS_KEY_ID: "XXXXXXXXX"
```

Comment:
These values should be read from a `env_file` that is added to gitignore. Storing credentials in files is an easy way to commit secrets.